ipi/inputs/system.py

```diff
@@ -8,6 +8,7 @@
 import os.path
 import sys
 import re
+import time
```

Comment:
Not used.

Reply:
there are many of those.... I'd let them be in case we need to time again. until we have proper timing mechanism :-)